app/controllers/api/v1/list_restaurants_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::ListRestaurantsController < ApplicationController
+  def index
+    restaurants = Restaurant.all
+    render json: {entries: restaurants}
```

Comment:
`ListRestaurantsController` sets a narrow scope for this controller. I would suggest `RestaurantsController`